Remotion/ObjectBinding/Web/UI/Controls/BocBooleanValueImplementation/Rendering/BocBooleanValueRenderer.cs

```diff
@@ -269,11 +269,11 @@ private void RegisterStarupScriptIfNeeded (BocBooleanValueRenderingContext rende
       scriptBuilder.Append (", ");
       scriptBuilder.Append (requiredFlag);
       scriptBuilder.Append (", ");
-      AppendStringValueOrNullToScript (scriptBuilder, renderingContext.Control.TrueDescription);
+      AppendStringValueOrNullToScript (scriptBuilder, renderingContext.Control.TrueDescription.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks));
```

Comment:
please add a ticket for adding an overload for this. (fyi: we already have a ticket for EscapeClientStript)